content/english/sql_workshop/activity2/_activity2.css

```diff
@@ -0,0 +1,76 @@
+/* Positions and style for the code blocks */

```

Comment:
To avoid duplication as suggested, I changed _activity1.css and _activity1.js to _default.css and _default.js and moved them to a global folder labeled "default" that way all activities will have access to them. For the activities that have slight alterations in the style and javascript code, I took out the code blocks that were exactly the same as the ones in the _default.files so the only code left in _activity2.css and _activity2.js should be the code that are (slightly) different to suit the activities' needs.